src/terminal7.js

```diff
@@ -1068,4 +1030,22 @@ export class Terminal7 {
         this.lastIdVerify = now
         return this.keys
     }
+    async getDotfile() {
+        return (await Preferences.get({key: "dotfile"})).value || DEFAULT_DOTFILE
+    }
+    saveDotfile() {
```

Comment:
I'm not sure how easy it is, but its much nicer to pass the conf as a parameter and let the function update `this.conf`

Reply:
We already have `loadConf` for updating `this.conf`, but maybe we should change `saveDotfile` to `loadAndSaveConf` (though it doesn't fit well imo).